daml/SwiftMessage/Util.daml

```diff
@@ -42,4 +42,10 @@ data PayoutType =
   deriving (Eq, Show)
 
 getRedemptionDate : Date -> Date -> Optional Date
-getRedemptionDate maturityDate payoutDate = if payoutDate == maturityDate then Some payoutDate else Nothing
\ No newline at end of file
+getRedemptionDate maturityDate payoutDate = if payoutDate == maturityDate then Some payoutDate else Nothing
```

Comment:
The canonical name for the negative case is `None`, I wasn't aware that `Nothing` also exists. Or are you defining an alias somewhere?

Reply:
Thanks for the catch, I didn't define it anywhere else.
Not sure how does it type check.
I will update this.

Reply:
updated